### DISCUSSION TITLE
Using elegoo cura 4.8 Neptune 2

### DISCUSSION THREAD

**Discussion by bnsf51atsf** [5axes/Calibration-Shapes#117] [Q&A]
I have followed the steps to update the plugin>resources>calibration. Shapes scripts and restarted cura. Still not seeing three new scripts. Is the cura version not fit required level?

**Comment by 5axes** [comment]
4.8 (versions prior to 4.9)  you need to use the function copy script.
![image](https://user-images.githubusercontent.com/11015345/190616020-12001d8a-fd4f-4f34-a25b-721a7413e209.png)

**Reply by bnsf51atsf** [reply]
I solved my issue by installing Utilmaker cura 5.1. Was able to get my Elegoo Neptune 2 working just fine with temp tower test and post script seting with temp fan script.
